lam4-backend/gf-grammar/updateHS.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Wait, what's `updateHS.sh` for? Is it something we could do in Haskell instead?

Reply:
updateHS.sh is running the GF executable to regenerate the Haskell code and the compiled PGF every time I change the GF grammar. Doesn't matter to me where it's run, just that it needs to have access to GF executable and not just the [PGF library](https://hackage.haskell.org/package/gf-3.11/docs/PGF.html).